ballerina/rag.bal

```diff
@@ -47,14 +48,16 @@ public distinct isolated class VectorRetriever {
     # Retrieves relevant chunks for the given query.
     #
     # + query - The text query to search for
+    # + topK - The maximum number of most similar items to return. Defaults to 10
```

Comment:
Do we also need to document the behaviour when -1 is provided here?

Reply:
fixed [53bca5c](https://github.com/ballerina-platform/module-ballerina-ai/pull/72/commits/53bca5ccc66a9536c8a4fcdf6e450b61bc68395c)